test/stub/service.go

```diff
@@ -0,0 +1,83 @@
+package stub
```

Comment:
if this is just for testing, you can also use `minimock` to generate mocks which you can use in your test (and you can customize the functions for each test as well).
See:
https://github.com/fabric8-services/fabric8-auth/blob/master/Makefile#L148-L160 + https://github.com/fabric8-services/fabric8-auth/blob/master/Gopkg.toml#L49 for the setup and code generation, then https://github.com/fabric8-services/fabric8-auth/blob/master/test/witservice.go#L14-L30 for using the mocks

Reply:
> if this is just for testing, you can also use minimock to generate mocks which you can use in your test (and you can customize the functions for each test as well).

for now, it is only for tests. In the future, I have a plan to use it for running tenant service locally.

Reply:
@MatousJobanek I understand your point. but does it mean we'll be removing this stub and asking everyone to run cluster locally before running this integration tests. I think we should introduce `minimock` to avoid more maintenance in future

Reply:
I don't follow @dipak-pawar

> but does it mean we'll be removing this stub

why removing this stub?

> asking everyone to run cluster locally before running this integration tests.

you need to run minishift to make the test running for sure - this is the intend of the test